docs/configuration/identity-providers/oidc.md

```diff
@@ -89,91 +89,103 @@ for which stage will have each feature, and may evolve over time:
+
+You must generate this option yourself. To generate a random string of sufficient length, you can use `openssl rand -base64 32`. If you are deploying this as a Kubernetes secret, you must encode it with base64 again, i.e. `openssl rand -base64 32 | base64`. Using secrets is always recommended.
+
+##### [6] authorization_policy
+
+The authorization policy for this client: either `one_factor` or `two_factor`.
+
+##### [7] redirect_uris
 
-The shared secret between Authelia and the application consuming this client. Currently this is stored in plain text.
+A list of valid callback URL´s this client will redirect to. All other callbacks will be considered unsafe. The URL's are case-sensitive. This differs from application to application - we have provided a list of URL´s for common applications below. If you do not find the application in the list below, you will need to search for yourself - and maybe come back to open a PR to add your application to this list so others won't have to search for them.
 
-#### authorization_policy
+`<DOMAIN>` needs to be substituted with the your domain and subdomain the application runs on. If GitLab, as an example, was reachable under `https://gitlab.example.com`, `<DOMAIN>` would be `gitlab.example.com`.
 
-The authorization policy for this client. Either `one_factor` or `two_factor`.
+| Application | Version              | Callback URL                                             |
+| :---------: | :------------------: | :------------------------------------------------------: |
+| GitLab      | `14.0.1`             | `https://<DOMAIN>/users/auth/openid_connect/callback`    |
+| MinIO       | `RELEASE.2021-06-17` | `https://<DOMAIN>/minio/login/openid`                    |
 
-#### redirect_uris
+##### [8] scopes
 
-A list of valid callback URL's this client will redirect to. All other callbacks will be considered unsafe. The URL's
-are case-sensitive.
+A list of scopes to allow this client to consume. See [scope definitions](#scope-definitions) for more information. The documentation for the application you want to use with Authelia will most-likely provide you with the scopes to grant.
 
-#### scopes
+##### [9] grant_types
 
-A list of scopes to allow this client to consume. See [scope definitions](#scope-definitions) for more information.
+A list of grant types this client can return. _It is recommended that this isn't configured at this time unless you know what you're doing_.
 
-#### grant_types
+##### [10] response_types
 
-A list of grant types this client can return. It is recommended that this isn't configured at this time unless you know
-what you're doing. 
+A list of response types this client can return. _It is recommended that this isn't configured at this time unless you  know what you're doing._
 
-#### response_types
+## Currently Tested Applications
```

Comment:
Same comment as above regarding the matrix of callbacks.

Reply:
👍🏼